dts/bindings/mtd/microchip,ateccx08-base.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Vogl Electronic GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+include: ["eeprom-base.yaml", i2c-device.yaml]
+
+bus: ateccx08
+
+properties:
+  wake-delay:
+    type: int
+    description: delay in us to wait for the device to wake up
+    default: 1500
```

Comment:
default values need justification in the description

Reply:
now added

Reply:
better to say "default value is the minimum according to the datasheet"